Controller/AjaxController.php

```diff
@@ -301,4 +304,57 @@ protected function getTokensAction(Request $request)
 
         return $this->sendJsonResponse($dataArray);
     }
+
+    /**
+     * @return \Symfony\Component\HttpFoundation\JsonResponse
+     *
+     * @throws \Exception
+     */
+    public function pendingEventsAction()
+    {
+        $objectId = $this->request->request->get('objectId') ? (int) ($this->request->request->get('objectId')) : null;
```

Comment:
A request within a request?

Reply:
The request object has a request object. It's weird.

Reply:
its like when you go to a restaurant and order off the menu, but you have to change everything about how its actually served pickles on the side, no mayo, add guac.